doc/userdoc/guides/nest2_to_nest3/whats_removed.rst

```diff
@@ -0,0 +1,81 @@
+  | ::                                          | ::                                    |
+  |                                             |                                       |
+  |     net = nest.LayoutNetwork(model, dim)    |     nrns = nest.Create(model, dim)    |
+  |     nrns = nest.GetLeaves(net)[0]           |                                       |
+  |                                             |                                       |
+  +---------------------------------------------+---------------------------------------+
+
+Printing the network as a tree of subnets is no longer possible. The
+``PrintNetwork()`` function has been replaced with ``PrintNodes()``, which
+prints ID ranges and model names of the nodes in the network.
+
+  +----------------------------------------------+---------------------------------------+
+  | NEST 2.x                                     | NEST 3.0                              |
+  +==============================================+=======================================+
+  |                                              |                                       |
+  | >>>  nest.PrintNetwork(depth=2, subnet=None) | >>>  nest.PrintNodes()                |
+  |      [0] root dim=[15]                       |      1 .. 10 iaf_psc_alpha            |
+  |      [1]...[10] iaf_psc_alpha                |      11 .. 15 iaf_psc_exp             |
+  |      [11]...[15] iaf_psc_exp                 |                                       |
+  |                                              |                                       |
+  |                                              |                                       |
+  +----------------------------------------------+---------------------------------------+
+
+Models
+~~~~~~
+
+With NEST 3.0, some models have been removed. They all have alternative models that can
+be used instead.
+
+  +----------------------------------------------+-----------------------------------------------+
+  | Removed model                                | Replacement model                             |
+  +==============================================+===============================================+
+  | iaf_neuron                                   | iaf_psc_alpha                                 |
+  +----------------------------------------------+-----------------------------------------------+
+  | aeif_cond_alpha_RK5                          | aeif_cond_alpha                               |
+  +----------------------------------------------+-----------------------------------------------+
+  | iaf_psc_alpha_presc                          | iaf_psc_alpha_ps                              |
+  +----------------------------------------------+-----------------------------------------------+
+  | iaf_psc_delta_canon                          | iaf_psc_delta_ps                              |
+  +----------------------------------------------+-----------------------------------------------+
```

Comment:
Please add the name change of `spike_detector` to `spike_recorder`

Reply:
done